app/views/organisation_reports/without_users_edit.html.erb

```diff
@@ -0,0 +1,24 @@
+<p><b> The Following email will be sent when a user is invited: </b></p>
+<div class="well" id="preview_email">
+  <%= render partial: 'shared/edit_mail_template', :locals => { :@resource => User.new } %>
+</div>
+<table class="table table-hover sortable">
```

Comment:
is this table here a repeat of what's in the without_users.html.erb template?  If so perhaps it should be DRYed out?

I guess there's also the question of what happens if the user clicks the invite button while the template is in edit mode ...?